### DISCUSSION TITLE
Better documentation around the build system

### DISCUSSION THREAD

**Discussion by daveverwer** [SwiftPackageIndex/SwiftPackageIndex-Server#1344] [Ideas]
I think there are two main issues that we need to solve with the build system documentation.

1. Visibility - The build system FAQ is hidden in the main site FAQ. Even though it's linked directly from the top of the build result pages, it could be far more visible.
2. Documentation for `.spi.yml` - This file is currently undocumented. 

This will probably need splitting into two tasks but this can serve as a starter issue for the task.

I think the documentation for the site needs to be a little bit more structured. I'm thinking a docs.swiftpackageindex.com with its own menu that has two sections (for now) "Overview" which has most of the current FAQ content, and "Build System" which has the build specific docs and the `.spi.yml` docs.

We should probably make this another static site, like the blog. I have a design in mind for it.

**Comment by finestructure** [comment]
I'm just adding watchOS support to my Parser lib and will write something up as I go through it.

**Comment by Sherlouk** [comment]
Unsure if relevant here, or it's own issue. But docs (https://swiftpackageindex.com/docs/builds) are out of date in places.

Mainly this line sticks out to me:

> Swift 5.3 (beta) builds with Xcode 12 (beta).

Given we support Swift 5.4 and are looking at 5.5, this is a bit old 😛
